packages/react-core/src/components/Select/SelectOption.tsx

```diff
@@ -209,22 +217,26 @@ export class SelectOption extends React.Component<SelectOptionProps> {
                 className={css(
                   styles.selectMenuWrapper,
```

Comment:
Hmm! The view more item should not change its background on hover, but this `.pf-c-select__menu-wrapper` class on the parent `<li>` is causing that background color to appear.

This class should only apply to an `<li>` if it is wrapping a favorites item - in that case, its children will be `.pf-c-select__menu-item.pf-m-link` (the favorite item text) and `.pf-c-select__menu-item.pf-m-action` (the favorite/star button). It looks like this line was added in https://github.com/patternfly/patternfly-react/pull/4642/files#diff-93c821c98c60ca88c0e8281f4ab9674ee65af75b0a4785592ab6f19dca05d6edR191.

We should remove it if possible. My only hesitation with removing it is a user may be targeting it with custom CSS since it's there. Maybe it would be OK to remove it and mention in the release notes that it was removed to fix a bug? WDYT?

Reply:
I updated it to not apply `.pf-c-select__menu-wrapper` when it is the view more button or the spinner.  The spinner however to still has has the background change on hover.

Reply:
@tlabaj looks like the spinner in https://patternfly-react-pr-5736.surge.sh/components/select/react/view-more/ and https://patternfly-react-pr-5736.surge.sh/components/select/react/view-more-with-checkboxes/ are loading differently. The item with the spinner should be appropriate for whatever kind of parent it's in (`<li>` if it's in a `<ul>`, `<div>` if it isn't) with the class `.pf-c-select__menu-item.pf-m-loading`. That will also center it, which you can see in the core docs - https://pf4.patternfly.org/components/select/#loading

Reply:
@mcoker So I updated the non checkbox select spinner.  The checkbox spinner parent element is a div, so I think that is right.  
I tired both `.pf-c-select__menu-item.pf-m-loading` and `.pf-c-select__list-item.pf-m-loading` for the checkbox  but neither style centered the spinner.

Reply:
@tlabaj hmm... it's not working now because it is `.pf-c-select__menu-item.pf-m-loading`, but changing it to `.pf-c-select__list-item.pf-m-loading` seems to work for me in the browser. 

`.pf-c-select__menu-item.pf-m-loading`
<img width="1703" alt="Screen Shot 2021-05-12 at 5 26 28 PM" src="https://user-images.githubusercontent.com/35148959/118051827-4b299b80-b347-11eb-80f1-643d61f1d14e.png">


`.pf-c-select__list-item.pf-m-loading`
<img width="1671" alt="Screen Shot 2021-05-12 at 5 24 40 PM" src="https://user-images.githubusercontent.com/35148959/118051666-00a81f00-b347-11eb-9e29-ef1d1f916fdc.png">